RULES_AND_GUIDELINES.md

```diff
@@ -0,0 +1,220 @@
+­­­Still in Progress­­­
+
+[^ back to top](#)
+
+## <a name="swamp coolers-and dryer installation/venting"></a>Swamp Coolers and Dryer Installation/Venting
+
+All are acceptable, at the owner's discretion. All, if any, must be installed exclusively inside an individual unit or fenced yard. None of these appliances, supports, wiring, hoses or other connections may be attached to the building exterior and no vents or exhausts may be installed within walls, floors, crawl spaces, attics or cut through building siding. Any of these violations may be corrected at any time without notice, at the board's discretion and all associated expenses will be the owner's sole responsibility.
+
+[^ back to top](#)
+
+## <a name="window and door repairs-and replacements"></a>Window and Door Repairs and Replacements 
+
+Repair and replacement of windows and doors are periodically necessary and are the responsibility of the unit owner. However, any repairs and replacements must be submitted to the board through an ARR and be in compliance with all of the following guidelines: All window and door frames must match the natural tones of the building or be painted to match. Size, shape and dimensions of doors and windows cannot be changed, moved or in any way alter the installed position within the building. Windows (including those on unit doors and storm doors) must be filled with a single pane of glass to maintain aesthetic cohesion with the property as a whole.
+
+[^ back to top](#)
+
+## <a name="Exterior Colors-and Paint"></a>Exterior Colors and Paint
+
+All exterior surfaces are subject to oversight by the board to assure aesthetic cohesion throughout the property. If paint is needed for a common area (e.g., the trim on a garage) it will be supplied, by request through the property manager at Hast & Co. If paint is needed for a limited common area (e.g., a white door or window) you or your contractor can get the paint directly from Sherwin Williams.
```

Comment:
I believe we mentioned that the paint will no longer be supplied by the property manager. I personally think it's fine to supply it as long as the resident returns the remaining unused paint.